src/TrackingOptOut.php

```diff
@@ -106,7 +106,7 @@ public static function renderSettings() {
   }
 
   public static function enqueueScriptsFirstHead() {
-    wp_enqueue_script(Plugin::PREFIX . '-google_analytics_opt_out', Plugin::getBaseUrl() . '/dist/scripts/google-analytics-opt-out.js');
+    wp_enqueue_script(Plugin::PREFIX . '-google_analytics_opt_out', Plugin::getBaseUrl() . '/dist/scripts/google-analytics-opt-out.js', FALSE);
```

Comment:
(and elsewhere) The 3rd argument to https://developer.wordpress.org/reference/functions/wp_enqueue_script/ is an array of dependencies, so adding a Boolean seems wrong here.